### PR TITLE
Updated "Learn more from the Cookbook" section

### PR DESCRIPTION
**Pull request by sfdumi**
Added missing links from the cookbook.

### CODE REVIEW

File: book/routing.rst [hunk 1]
```diff
@@ -1608,5 +1608,12 @@ Learn more from the Cookbook
 ----------------------------
 
 * :doc:`/cookbook/routing/scheme`
+* :doc:`/current/cookbook/routing/slash_in_parameter`
+* :doc:`/current/cookbook/routing/redirect_in_config`
+* :doc:`/current/cookbook/routing/method_parameters`
+* :doc:`/current/cookbook/routing/service_container_parameters`
+* :doc:`/current/cookbook/routing/custom_route_loader`
+* :doc:`/current/cookbook/routing/redirect_trailing_slash`
+* :doc:`/current/cookbook/routing/extra_information`
```
Comment (xabbuh) [inline book/routing.rst]: You have to remove the `/current` prefix here.

Comment (sfdumi) [inline book/routing.rst]: Thanks. Missed that.

**Comment by xabbuh** [comment]
:+1:

**Comment by WouterJ** [comment]
Thank you for updating this section @sfdumi! Seems like we haven't updated this list for a long time. I've merged your PR in the 2.3 branch of the docs and will merge it into the newer versions from there (this is the reason behind the closed status instead of the merged one).

**Pull request closed without merging**